app/controllers/api/v1/migrations/external_controller.rb

```diff
@@ -61,6 +61,30 @@ def create_user
           render_data status: :created
         end
 
+        # POST /api/v1/migrations/rooms.json
+        # Expects: { room: { :name, :friendly_id, :meeting_id, :last_session } }
+        # Returns: { data: Array[serializable objects] , errors: Array[String] }
+        # Does: Creates a room.
+        def create_room
+          room_hash = room_params.to_h
+
+          user = User.find_by(email: room_hash[:owner_email], provider: room_hash[:owner_provider])
+
+          return render_error status: :bad_request unless user
+
+          room = Room.new(room_hash.except(:owner_email, :owner_provider).merge({ user: }))
+
+          # Redefines the validations method to do nothing
+          # rubocop:disable Lint/EmptyBlock
+          room.define_singleton_method(:set_friendly_id) {}
+          room.define_singleton_method(:set_meeting_id) {}
```

Comment:
Good use of ruby that saves us from impacting our codebase because of this temporary feature,
However, having the meeting ids propagating from v2 makes sense but do we need the friendly ids?

Reply:
I guess we don't really need them, but might as well?

Reply:
That worked because we conserved somehow the same schema for both friendly ids on v2 and v3,
A cleaner approach is to pass only what v3 requires from v2 and generate what v3 can generate,

Reply:
My reasoning was that it would keep the URLs/bookmarks of the rooms

Reply:
Yeah the URLs will be different from v2. I can't think of a reason to keep the friendly_ids neither.

Reply:
Well, good reasoning.
However, that may not help since on v2 rooms are accessed directly on the root path when on v3 they're under `/rooms`.
Besides, most of v2 deployments are hosting greenlight under a root mainly `/b` or `/gl`,